tools/copydlldeps.md

```diff
@@ -45,6 +45,24 @@ if [ ! $( echo $compiler | grep -q "shared" ) ]; then
 fi
 ```
 
+Test & example
+==============
+
+Let us compile password manager [KeePassX](https://www.keepassx.org) in
+shared mode (i686-w64-mingw32.shared) and gather all its DLL dependencies.
+
+```bash
+$ make keepassx MXE_PLUGIN_DIRS=plugins/apps \
+    MXE_TARGETS='i686-w64-mingw32.shared'
+$ ./tools/copydlldeps.sh --copy --destdir keepassx-bundle \
+    --infile usr/i686-w64-mingw32.shared/bin/KeePassX.exe \
+    --recursivesrcdir usr/i686-w64-mingw32.shared/
+$ cp -r usr/i686-w64-mingw32.shared/bin/{KeePassX.exe,share} keepassx-bundle
```

Comment:
I mean follow commands in tools/copydlldeps.md which are being added by this PR.

E.g., to build keepassx, `make keepassx MXE_PLUGIN_DIRS=plugins/apps MXE_TARGETS='i686-w64-mingw32.shared'`. Building keepassx manually in MXE is hard and unneeded work.

> Review: every qt application seems to need
> --enforce usr/i686-w64-mingw32.shared/qt5/plugins/platforms/

I hope keepassx doesn't need it, as it uses Qt4, not Qt5.